### ISSUE TITLE
Changes in `pytorch_neural_network_simple.ipynb`

### ISSUE THREAD

**Issue by ChrisVilches** [ChrisVilches/Machine-Learning#9]
- [x] turn it into Quarto
- [x] batch size is too small
- [x] `running_loss += loss.item()` is probably wrong (must audit)
- [x] `running_loss/len(train_loader)` is probably wrong (must audit)

Check the other neural network I made (fashion mnist) to see the differences. Also, there's the probability that this file is correct, and fashion mnist is the wrong one. Must verify all of that.
